sql3/planner/opinsert.go

```diff
@@ -336,7 +336,25 @@ func (i *insertRowIter) Next(ctx context.Context) (types.Row, error) {
 						return nil, errors.Wrapf(err, "converting timestamp to int64: %s", v)
 					}
 					row.Values[posVals[idx]] = i64
-
+				//integers passed as input for Timestamp fields will be treated as time represented in number of seconds since unix epoch
+				case int64:
+					// Convert the input seconds to target timeunit defined for the Timestamp field
+					// Add Base, which is the base epoch for Timestamp fields, to the input before saving
+					unit := fbbatch.TimeUnit(opts.TimeUnit)
+					i64 := opts.Base
```

Comment:
the comments say "seconds since unix epoch", but it looks like here we're using "opts.Base", which might not be the Unix epoch.

Reply:
Comment is updated to say fields can have custom epoch, but fields created using SQL will default to unix epoch.